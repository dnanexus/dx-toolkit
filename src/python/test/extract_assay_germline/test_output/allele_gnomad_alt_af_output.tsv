allele_id	chromosome	starting_position	ref	alt	rsid	allele_type	dataset_alt_freq	gnomad_alt_freq	worst_effect
