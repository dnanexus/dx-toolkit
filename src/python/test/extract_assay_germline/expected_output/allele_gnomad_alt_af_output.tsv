allele_id	chromosome	starting_position	ref	alt	rsid	allele_type	dataset_alt_freq	gnomad_alt_freq	worst_effect
21_8986952_T_C	21	8986952	T	C	rs1344912965	SNP	0.0	0.00113037	
